roda-ui/roda-wui/src/main/java/org/roda/wui/client/common/slider/InfoSliderHelper.java

```diff
@@ -149,13 +151,13 @@ public static void updateInfoSliderPanel(BrowseRepresentationBundle bundle, Slid
       values.put(messages.representationType(), createRepresentationTypeHTML(bundle));
     }
 
-    if (StringUtils.isNotBlank(representation.getInstanceId())) {
-      values.put(messages.itemInstanceId(), createRepresentationInstanceIdHTML(bundle));
-    }
+    String instanceId = representation.getInstanceId();
+    addLinkifCentralInstance(values, bundle.getInstanceName(), instanceId);
```

Comment:
You can write this in one line, make these changes to the other bundles too
```suggestion
    addLinkifCentralInstance(values, bundle.getInstanceName(), representation.getInstanceId());
```

Reply:
Done